package.json

```diff
@@ -73,9 +73,11 @@
         "@netgen/javascript-cookie-control": "^0.0.8",
         "@popperjs/core": "^2.11.6",
         "bootstrap": "^5.3.3",
+        "flatpickr": "^4.6.13",
```

Comment:
Didn't I see these changes in your other PR?

Reply:
Yes, rebase error. Fixed.